fiftyone/utils/sam3.py

```
@@ -545,3 +551,738 @@ def _forward_pass_auto(self, imgs):
+                            }
+                        )
+                    if frame_prompts:
+                        prompts[frame_idx] = frame_prompts
+
+                else:
+                    raise ValueError(
+                        f"Unsupported prompt type {type(value)} on frame "
+                        f"{frame_number}. Use Detections or Keypoints. "
+                    )
+
+            return prompts
```

Ruff (0.15.15)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)